CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,57 @@
+
+This code of conduct applies to all spaces and events managed by the Freedom of the Press Foundation. This includes physical offices, GitHub repositories, online chat systems, the Support Forum, FPF hosted or sponsored events, and any other forums created by the Foundation which our team uses for communication. In addition, we take violations of this code outside these spaces into account when determining a person's ability to participate within them.
+
+If you believe someone is violating the code of conduct, we ask that you report it by talking to Emmanuel, or emailing **emmanuel@freedom.press**. If, for whatever reason, you are uncomfortable reporting the issue to Emmanuel, you may contact FPF’s Executive Director (**trevor@freedom.press**).  
+
+We are committed to taking fair, timely and appropriate corrective action against any violations of this code.
+
+* **Be friendly and patient.**
+
+- **Be welcoming.** We strive to be a workplace that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+
+- **Be kind and considerate.** Your work will be used by other people, and you in turn will depend on the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we work with people as part of an international community, so you might not be communicating in someone else's primary language.
+
+- **Be respectful.** Not all of us will agree all the time, but disagreement is no excuse for poor behavior or poor manners. We might all experience some frustration now and then, but we cannot allow that frustration to turn into a personal attack. It’s important to remember that a workplace where people feel uncomfortable or threatened is not a productive one. Freedom of the Press Foundation staff should be respectful when dealing with other staff as well as with people outside the Foundation, such as clients, outside contributors, and the SecureDrop community. Avoid making assumptions, e.g., about people’s gender identity, preferred pronoun, willingness to be photographed, etc.—when in doubt, ask at the right moment.
+
+- **Harassment, public or private, is unacceptable and won’t be tolerated.** This includes but is not limited to:
+  - Intimidation
+  - Stalking
+  - Unwelcome following
+  - Enlisting the help of others, whether in person or online, in order to target an attendee
+  - Taking photographs, video, or audio recordings or recordings without consent
+  - Obscene or intimidating gestures
+  - Shouting
```

Comment:
> Shouting

This feels too vague for me. @msheiny and I like to act ridiculous and squabble in all caps, and that's ok. I think it would be better to roll this up into the previous and say:

> Obscene or intimidating gestures or tone, either written or spoken

For example shouting "I'll fight you" in an silly debate about tabs/spaces could be fine if the tone is obviously light and the participants know each other. Saying "I'll fight you" after someone asks you to use a different pronoun with them is probably never ok.

Reply:
+1 to the modified language. I believe the original was intended to refer to spoken language, but I think your clarified wording is perfect.

Reply:
+1 as well. Thanks for that clarification, @heartsucker